@types/webGPU.d.ts

```diff
@@ -0,0 +1,3232 @@
+// *********************************************************************************************
+// This file is manually-edited by diffing against an autogenerated file. See README.md.
```

Comment:
How is this file generated?

Reply:
This is a direct quote from the official project.

Reply:
Can we use `npm install` to add these dependencies? Instead of add them manually.

Reply:
I need to look into this further.